src/cmd/linuxkit/build.go

```diff
@@ -0,0 +1,215 @@
+package main
```

Comment:
This file is more or less a verbatim copy of https://github.com/moby/tool/blob/master/cmd/moby/build.go and I'm not too keen on the duplication...

Reply:
It is identical, not more-or-less, so it is easy to update. Not ideal, maybe can move some more code in to the library later.

Reply:
OK, identical is better than almost identical. Having this moved to some common piece of code somewhere would be good, but could be done later